playground/infrastructure/datastore_client.py

```diff
@@ -271,10 +270,12 @@ def _to_example_entity(self,
                 "descr": example.tag.description,
                 "tags": example.tag.tags,
                 "cats": example.tag.categories,
-                "path": example.link,
+                "path": example.url_vcs, # keep for backward-compatibity, to be removed
                 "type": PrecompiledObjectType.Name(example.type),
                 "origin": origin,
-                "schVer": schema_key
+                "schVer": schema_key,
+                "urlVCS": example.url_vcs,
+                "urlNotebook": example.url_notebook,
```

Comment:
Can you check the Python's habits for acronyms? In Dart and many other languages the habit is to capitalize only the first letter. For instance the class for URI is named `Uri` in Dart. I guess `urlVcs` would better in this case.

And also since we change to 'VCS URL' order this becomes `vcsUrl` and `notebookUrl`.

Reply:
this casing is already used in other Datastore fields in Playground